# airconv: DXBC's sample_c has no 1D case again, and writes nothing
diff --git a/src/airconv/nt/dxbc_converter_base.cpp b/src/airconv/nt/dxbc_converter_base.cpp
index ec34c22..8b9b4a3 100644
--- a/src/airconv/nt/dxbc_converter_base.cpp
+++ b/src/airconv/nt/dxbc_converter_base.cpp
@@ -2037,16 +2037,6 @@ Converter::operator()(const InstSampleCompare &sample) {
   llvm::Value *ArrayIndex = nullptr;
 
   switch (Tex->Logical) {
-  case Texture::texture1d:
-    Coord = LoadOperand(sample.src_address, kMaskComponentX);
-    Coord = ir.CreateInsertElement(llvm::ConstantFP::getNullValue(air.getFloatTy(2)), Coord, 0ull);
-    break;
-  case Texture::texture1d_array:
-    Coord = LoadOperand(sample.src_address, kMaskComponentX);
-    Coord = ir.CreateInsertElement(llvm::ConstantFP::getNullValue(air.getFloatTy(2)), Coord, 0ull);
-    ArrayIndex = LoadOperand(sample.src_address, kMaskComponentY);
-    ArrayIndex = ClampArrayIndex(ArrayIndex, Tex->Metadata);
-    break;
   case Texture::depth2d:
     Coord = LoadOperand(sample.src_address, kMaskVecXY);
     break;
